packages/core/src/utils/react-query-utils.ts

```diff
@@ -146,7 +147,10 @@ export const retryFunction = (failureCount: number, error: any) => {
   return false
 }
 
-export const defaultQueryConfig = {
-  suspense: true,
-  retry: retryFunction,
+export function getDefaultQueryConfig() {
+  const {suspenseEnabled} = getBlitzRuntimeData()
+  return {
+    suspense: !!suspenseEnabled,
+    retry: retryFunction,
+  }
```

Comment:
We should turn this into a hook and use `useMemo`. Dependency array will be `[]` because it will never change

Reply:
Done.